stdlib/public/core/ArrayBuffer.swift

```diff
@@ -101,7 +101,7 @@ extension _ArrayBuffer {
 
   /// Returns `true` iff this buffer's storage is uniquely-referenced.
   @inlinable
-  internal mutating func isUniquelyReferenced() -> Bool {
+  internal func isUniquelyReferenced() -> Bool {
```

Comment:
Isn't this change ABI breaking?

Reply:
How is CoW supposed to work if we're checking uniqueness on a copy of 'self'? This all needs to work without any optimization kicking in.